run_nodes.sh

```diff
@@ -1,45 +1,78 @@
 #!/bin/bash
 
-if [ -z "$1" ] || (("$1" < 2 || "$1" > 8))
-then
-    echo "The committee size is missing, usage:
+function usage(){
+  echo "Usage:
+      ./run_nodes.sh [-v N_VALIDATORS] [-n N_NON_VALIDATORS] [ALEPH_NODE_ARG]...
+  where 2 <= N_VALIDATORS <= N_VALIDATORS + N_NON_VALIDATORS <= 10
+  (by default, N_VALIDATORS=4 and N_NON_VALIDATORS=0)"
+}
 
-    ./run_nodes.sh SIZE [Additional Arguments to ./target/debug/aleph-node]
+N_VALIDATORS=4
+N_NON_VALIDATORS=0
 
-where 2 <= SIZE <= 8"
-    exit
-fi
+while getopts "v:n:" flag
+do
+  case "${flag}" in
+    v) N_VALIDATORS=${OPTARG};;
+    n) N_NON_VALIDATORS=${OPTARG};;
+    \?)
+      usage
+      exit
+      ;;
+  esac
+done
+
+shift $((OPTIND-1))
 
 killall -9 aleph-node
 
 set -e
 
 clear
 
-n_members="$1"
-echo "$n_members" > /tmp/n_members
-shift
+cargo build --release -p aleph-node
```

Comment:
this could also be a flag, that defaults to true, else LGTM

Reply:
done